internal/dinosql/parser.go

```diff
@@ -340,20 +340,34 @@ func validateQueryName(name string) error {
 		isLetter := unicode.IsLetter(c) || c == '_'
 		isDigit := unicode.IsDigit(c)
 		if i == 0 && !isLetter {
-			return fmt.Errorf("invalid query name: %q", name)
+			return fmt.Errorf("invalid query name %q", name)
 		} else if !(isLetter || isDigit) {
-			return fmt.Errorf("invalid query name: %q", name)
+			return fmt.Errorf("invalid query name %q", name)
 		}
 	}
 	return nil
 }
 
-func parseMetadata(t string) (string, string, error) {
+type CommentStyle int
+
+const (
+	MySQLCommentStyle CommentStyle = iota
```

Comment:
Instead of naming these after the database engine, let's name them after the comment syntax

```go
type CommentSyntax int

const (
    CommentSyntaxDash CommentSyntax = iota
    CommentSyntaxStar
    CommentSyntaxHash
)
```

Also, note that the type name comes first in the constant name. This is consistent with the [HTTP status constants](https://golang.org/src/net/http/status.go) and the errors in the [os package](https://golang.org/pkg/os/#pkg-variables).

Reply:
Nice catch.